### ISSUE TITLE
Speaker notes not working through pip

### ISSUE THREAD

**Issue by Jegp** [damianavila/RISE#497]
Hey, first of all thanks for a great project!

I've recently run into the following issue when attempting to use speaker notes. I've reproduced it using vanilla installations through `pip`. Most recently with jupyter `1.0.0`, jupyter notebook `6.0.0` and rise `5.5.1`.

When I start a slideshow and press `t`, nothing happens and I consistently get this warning:

`[W 20:43:41.060 NotebookApp] 404 GET /nbextensions/plugin/markdown/marked.js (127.0.0.1) 1.15ms referer=http://localhost:8888/nbextensions/rise/notes_rise/notes.html`

I tried to debug it, but I wanted to hear your opinion. I assume it's a problem with nbextensions somewhere. I installed all extensions with `jupyter contrib nbextension install --sys-prefix` (I'm in a virtualenv) and made sure they're listed in `jupyter nbextension list`. Full log is attached here:

[jupyter.log](https://github.com/damianavila/RISE/files/3414941/jupyter.log)

**Comment by damianavila** [comment]
> I assume it's a problem with nbextensions somewhere. I installed all extensions with jupyter contrib nbextension install --sys-prefix (I'm in a virtualenv) and made sure they're listed in jupyter nbextension list

So you are using the contrib extension alongside with RISE, right?
Can you try with a clean env without those extensions and check for the behaviour?
In that way, if you don't see the issue, we can start thinking about some sort of conflict between the extension...

**Comment by Jegp** [comment]
I tried to reproduce the issue, but failed. I used clean environments with fixed versions (jupyter 1.0.0, jupyter notebook 6.0.0 and rise 5.5.1), so I'm puzzled why it suddenly works again. It could be related to my windowmanager (i3wm), although I don't see any odd behaviour in the logs. I would assume the issue is closed, although it's bugging me what happened. Thanks for getting back to me @damianavila!
